frontend/src/components/common/table/table/constants.scss

```diff
@@ -0,0 +1,6 @@
+$table-background-color: #242636;
+$table-head-text-color: #ffffff;
+$line-under-head-color: #393b4d;
+$table-rows-text-color: #6b6d7f;
+$table-head-font: roboto, sans-serif;
+$table-body-font: inter, sans-serif;
```

Comment:
I mean create declare these variables in `src/assets/css/variables.scss` - as they will be used everywhere for styling